### ISSUE TITLE
Rename Archaeological Permit to HCA Permit

### ISSUE THREAD

**Issue by bferguso** [bcgov/nr-bcap#637]
As per a message from @maria-e-schmidt today, the resource name for permits should be changed from `Archaeological Permit` to `HCA Permit`. Need to:
- [x] Update the graph name and graph slug for the above change
- [x] Update the graph nodes and widget labels in resource graphs that reference it 
- [x] Update ETL to use the new names
